schema/schema.cql

```diff
@@ -24,3 +24,9 @@ create table metric_name_set (
   metric_names set<varchar>,
   primary key (shard)
 );
+
+-- graphite_names
+create table graphite_names (
```

Comment:
i think we should do something similar as the metric_names. querying this would be quite inefficient, and I rather have some shard key to shard data around.